### PR TITLE
Refractored the Cassandra Store for cleanup. Removed a function and m…

### PR DESCRIPTION
**Pull request by jzeeck**
…ade a future that checked for duplicates in the databse. Convers a ResultSet to a List of Row

### CODE REVIEW

File: cassandra/src/main/java/se/tre/freki/storage/cassandra/CassandraStore.java [hunk 1]
```diff
@@ -602,8 +549,11 @@ protected ResultSetFuture fetchTimeSeriesPartition(final ByteBuffer timeSeriesId
   ListenableFuture<List<String>> getNames(final LabelId id,
                                           final LabelType type) {
 
-    final ListenableFuture<List<Row>> namesFuture = fetchLabelRows(id, type,
-        getNameStatement.bind(toLong(id), type.toValue()));
+    BoundStatement getNameBoundStatement = getNameStatement.bind(toLong(id), type.toValue());
+
+    final ListenableFuture<List<Row>> namesFuture = transform(
+        session.executeAsync(getNameBoundStatement),
+        new ResultSetToRowListFunction("Found duplicate IDs for ({}, {})", id, type));
```
Comment (luuse) [inline cassandra/src/main/java/se/tre/freki/storage/cassandra/CassandraStore.java]: Can you replace `IDs` with `names` in this log message?

File: cassandra/src/main/java/se/tre/freki/storage/cassandra/functions/ResultSetToRowListFunction.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class ResultSetToRowListFunction implements Function<ResultSet, List<Row>> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ResultSetToRowListFunction.class);
+
+  private String message;
+  private Object[] param;
```
Comment (luuse) [inline cassandra/src/main/java/se/tre/freki/storage/cassandra/functions/ResultSetToRowListFunction.java]: Can you make these fields final?

File: cassandra/src/main/java/se/tre/freki/storage/cassandra/functions/ResultSetToRowListFunction.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.google.common.base.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class ResultSetToRowListFunction implements Function<ResultSet, List<Row>> {
```
Comment (luuse) [inline cassandra/src/main/java/se/tre/freki/storage/cassandra/functions/ResultSetToRowListFunction.java]: Not sure if it's better but maube `MultipleRowLoggingFunction`? Can you add some javadoc in any case?

**Comment by jzeeck** [comment]
See #95

**Pull request merged**